README.md

```diff
@@ -55,3 +62,58 @@ Use:
 <!-- Use the icon! -->
 <i class="af pencil"></i>
 ```
+
+## Vite
+
+From version `v1.0.0` you can use `svg-packer` with Vite:
```

Comment:
we should add a note here about the dependency required to use the Vite plugin, `vite-plugin-node-polyfills` as dependency but will require also `node-stdlib-browser`: in my external tests with Vite using `pnpm` I needed to add `shamefully-hoist=true` in the `.npmrc`, maybe we can just add `node-stdlib-browser` to dependencies or just move both to peer dependencies with optional.